app/screens/edit_profile/edit_profile.tsx

```diff
@@ -120,19 +137,110 @@ const EditProfile = ({
         setCanSave(value);
     }, [componentId, rightButton, isTablet]);
 
+    // Get database and observables
+    const database = DatabaseManager.getServerDatabaseAndOperator(serverUrl).database;
+    const dbAttributes = useState(() =>
+        (enableCustomAttributes ? observeCustomProfileAttributesByUserId(database, currentUser?.id || '') : null),
+    )[0];
+    const dbFields = useState(() =>
+        (enableCustomAttributes ? observeCustomProfileFields(database) : null),
+    )[0];
```

Comment:
I don't like this, should be done `withObservables` higher component that does take care of unsubscribing, this will lead to a memory leak.